pennylane/devices/default_qutrit.py

```diff
@@ -0,0 +1,261 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+The default.qutrit device is PennyLane's standard qutrit-based device.
+
+It implements the :class:`~pennylane._device.Device` methods as well as some built-in
+:mod:`qutrit operations <pennylane.ops.qutrit>`, and provides simple pure state
+simulation of qutrit-based quantum computing.
+"""
+import functools
+import numpy as np
+
+import pennylane as qml  # pylint: disable=unused-import
+from pennylane import QutritDevice
+from pennylane.wires import WireError  # pylint: disable=unused-import
+from .._version import __version__
+
+# tolerance for numerical errors
+tolerance = 1e-10
+
+OMEGA = np.exp(2 * np.pi * 1j / 3)
+
+
+class DefaultQutrit(QutritDevice):
+    """Default qutrit device for PennyLane.
+
+    Args:
+        wires (int, Iterable[Number, str]): Number of subsystems represented by the device,
+            or iterable that contains unique labels for the subsystems as numbers (i.e., ``[-1, 0, 2]``)
+            or strings (``['ancilla', 'q1', 'q2']``). Default 1 if not specified.
```

Comment:
Are there already hardware devices with these naming conventions?

Reply:
Not as far as I'm aware. Most of the functions implemented here (along with their documentation, as is the case here) are extremely heavily inspired by `DefaultQubit`.